lang/en/docs/models/auxiliary-concepts/esm.md

```diff
@@ -0,0 +1,24 @@
+# Effective Screening Medium
+
+The **Effective Screening Medium (ESM)** [^1] is a computational scheme allowing to simulate **surfaces** and **interfaces**, which are periodic in the direction parallel to the surface, and of arbitrary shape in the perpendicular direction. The method was developed and originally described in Ref. [^2].
+
+The  surface/interface slab system (the "working" region) is typically sandwiched between semi-infinite media, such as vacuum, an electrode, or an electrolyte. Here, we call this latter medium the "effective screening medium" (ESM), such as represented in the image below.
+
+![Editor](../../images/models/esm.png "Editor")
+
+The ESM model affords for the modeling of polarized surfaces without dipole correction.
+
+## Applications of ESM
+
+This ESM method can be used to calculate the total energy, charge density, force, and potential of a polarized or charged slab (which can also consist in molecules and clusters).
+
+ESM screens the electronic charge of a polarized/charged medium along one perpendicular direction by introducing a classical charge model and a local relative permittivity into the first-principles calculation framework. 
+
+## Implementation
```

Comment:
This page should have a section about the "Boundary Conditions" and refer there to the the other page with explanation